python/sqlalchemy/src/config.py

```diff
@@ -0,0 +1,16 @@
+import logging
+
+
+listen_port = 8080
+db_user = 'postgres'
+db_password = None
+database = 'ysql_sqlalchemy'
+schema = 'ysql_sqlalchemy'
```

Comment:
Same about the schema here (database is good) though these changes seem to be already in master. Maybe a pull/rebase is needed?